extension/src/popup/components/sendPayment/SendAmount/index.tsx

```diff
@@ -65,6 +66,41 @@ const ConversionRate = ({
   </div>
 );
 
+const BalanceOption = ({
+  balance: [key, balance],
+}: {
+  balance: [string, Types.AssetBalance | Types.NativeBalance];
+}) => {
+  const [assetDomain, setAssetDomain] = useState("stellar.org");
+  const assetIssuer = "issuer" in balance.token ? balance.token.issuer.key : "";
+  const networkDetails = useSelector(settingsNetworkDetailsSelector);
+  const server = new StellarSdk.Server(networkDetails.networkUrl);
+
+  useEffect(() => {
+    const fetchAssetDomain = async () => {
+      let homeDomain = "";
+      // https://github.com/stellar/freighter/issues/410
+      try {
+        ({ home_domain: homeDomain } = await server.loadAccount(assetIssuer));
+      } catch (e) {
+        console.error(e);
+      }
+
+      setAssetDomain(homeDomain);
+    };
+
+    if (balance.token.type !== "native") {
+      fetchAssetDomain();
+    }
+  }, [assetIssuer, server, balance.token.type]);
+
+  return (
+    <option key={key} value={key}>
+      {balance.token.code} &bull; {assetDomain}
```

Comment:
💭  thoughts on no bullet if no assetDomain?
```suggestion
      {balance.token.code} {assetDomain && "&bull;"} {assetDomain}
```